docs/adrs/0002-python-django-postgres.md

```diff
@@ -0,0 +1,37 @@
+# Architecture decision record 2: Python, Django, PostgreSQL
+
+The tech stack we selected for replatforming DARTS
+
+* :white_check_mark: _Status:_ Implemented [December 2024](https://github.com/DOI-OS-ORDA/DARTS/commit/1858055b77cc9f840480fb4f9ddfe0314e0992ec)
```

Comment:
clarification: Are we writing this one up as if it's already decided? Or did we want to at least go through the decision-making process to formalize it? I can see value either way.

Reply:
I updated with today's meeting